contracts/proxy/Clones.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: MIT
+// COPIED FROM https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.3.0/contracts/proxy/Clones.sol
```

Comment:
I diff checked against OZ's Clones and confirmed this is (nearly) identical. Any reason we're not just importing Clones from OZ? Is it just the `pragma solidity` compiler version?